OneCommunityGlobal/HighestGoodNetworkApp
# Ramakrishna _Allowing the numbers in teamcode

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/19d64bdf-0150-4339-9569-4a25687aaeb3)


## Related PRS (if any):
This frontend PR is related to the [#979](https://github.com/OneCommunityGlobal/HGNRest/pull/979) backend PR.

…

## Main changes explained:
-updated the regex the for teamcode changes in formatted.jsx, teamtable.jsx,userProfile.jsx,userTeamsTable.jsx,SaveButton.jsx
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin/owner user or any user who can edit teamcodes 
5. Go Reports->weekly Summaries Report 
6. In this page Try to edit the teamcode and check whether you're able to add numbers in teamcode and also check whether the teamcode pass this conditions : length of team code is between 5 to 7 
                                                         A-AAAAA or AAAAAA or A-1AAAA or A12AAAA
7. Now check the same thing in profile page where teamcode can be added 
video explainations:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/0748ab94-7baf-4880-89db-da2b92d139da

##Note:
In weeklysummarie report page for some users while editing the teamcode we are getting 500 userprofile api error and that error is not related to this PR changes, so can you kindly change the teamcode which are able to change Thank you!
